src/main/java/hudson/plugins/parameterizedtrigger/BuildInfoExporterAction.java

```diff
@@ -243,40 +380,48 @@ public void buildEnvVars(AbstractBuild<?, ?> build, EnvVars env) {
    * @return
    */
   public Object readResolve() {
-    if (this.lastReference == null) {
-      this.lastReference = new BuildReference(this.buildName, this.buildNumber, Result.NOT_BUILT);
+    if (this.lastReference2 == null) {
+      this.lastReference2 = this.lastReference;
```

Comment:
@kwhetstone 
I made some tests an it seems like even the old code didn't work like expected. Do you know what the transient field buildRefs is supposed to do? I am asking because the resolveObject-method seems to expect that a transient field is restored if it wasn't transient before. However I made some tests and transient field aren't restored by java serialization nor by xstream.
transient before. However I made some tests and transient field aren't restored by java serialization nor by xstream.